api/src/main/java/org/pmiops/workbench/db/dao/ParticipantDao.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.db.dao;
+
+import org.pmiops.workbench.db.model.Participant;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Page;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface ParticipantDao extends CrudRepository<Participant, Long> {
+
+    Page<Participant> findParticipantByParticipantKey_CohortIdAndParticipantKey_CdrVersionId(@Param("cohortId") long cohortId,
```

Comment:
We should consider using Slice here instead of Page. The number of participants in the cohort will be written into the review row at cohort creation time, and won't change; we don't need JPA to look it up for us each time we fetch a page.